code/packaging/app-config/pom.xml

```diff
@@ -1,6 +1,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <artifactId>cattle-app-config</artifactId>
+    <dependencies>
```

Comment:
Out of curiosity, what does this solve by putting these here?  Does this help with IDE auto-completion?  This project is just config files, so there obviously is no real code dependencies.